CHANGELOG.md

```diff
@@ -1,3 +1,73 @@
+# v1.0.6 (2021-02-26)
+
+## OS Changes
+* Add metricdog support to send anonoymous bottlerocket metrics ([#1006], [#1322])
+* Add a vmware-dev variant ([#1292], [#1288], [#1290])
+* Add k8s static pods support ([#1317])
+* Add high-level 'set' subcommand for changing settings in apiclient ([#1278])
+* Add "shibaken" generator to populate user-data with public keys from IMDS ([#1331])
+* Add support for kubelet in standalone mode and TLS auth ([#1338])
+* Add https-proxy and no-proxy settings to updog ([#1324])
+* Add support for ECR Public to hostctr ([#1296])
+* Add network proxy support to k8s 1.19 ([#1337])
+* Add support for kmot kit ([#1287], [#1286], [#1285])
```

Comment:
Does this seem ok or sounds incomplete ?

Reply:
(typo: "kmod")

Maybe "Add support for kmod kits to ease building of third-party kernel modules"?